modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -312,11 +318,11 @@ static int maxLineConsumption(const cv::GFluidKernel& k, int inH, int outH, int
     }
 }
 
-static int borderSize(const cv::GFluidKernel& k)
+static int borderSize(const cv::GFluidKernel::Kind kind, int window)
```

Comment:
So why you decided to go away from `cv::GFluidKenrel&` object here to more specific fields? Doesn't that object hold the necessary `kind`/`window` information already?

Reply:
The fact is that now GFluidKernel doesn't contain a "window" field. According to Ruslan's suggestion, I moved the "window" field from GFluidKernel to a more specific and convenient for further use structure GFluidUnit. I think duplication is useless to us in this case. So, I was had to change signature of the borderSize() function.